lib/modules/manager/sbt/scala.ts

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Need to bump `good-enough-parser` dependency to `1.1.22` which contains the change you need

Reply:
will trigger update via dashboard

Reply:
- #19672